doc/supportedPlatformVerions.md

```diff
@@ -137,8 +137,13 @@
 - 14.13.1
 - 14.14.0
 - 14.15.0
+- 14.15.1
 - 15.0.0
 - 15.0.1
+- 15.1.0
+- 15.2.0
+- 15.2.1
+- 15.3.0
 
 ## php
 
```

Comment:
Now we are supporting different version of debian (stretch and buster) and ubuntu .. and all these versions are not in sync .. for example a buster php version is 7.4 but stretch php latest is 7.3 .. so these document should reflect which debian type (read buster or ubuntu or stretch) supports which platform version.

Lets discuss this on Monday

Reply:
yeah agree, we should also list out supported buster versions and ubuntu versions. I'll it update here.